src/coreanimation.cs

```diff
@@ -1765,9 +1762,6 @@ interface CAEmitterBehavior : NSSecureCoding {
 		[Export ("type")]
 		string Type { get; }
 
-		[Static][Export ("behaviorTypes")]
```

Comment:
There's a comment for the class that should probably be updated (although we should probably remove the entire class)